src/index.css

```
@@ -1,50 +1,368 @@
+/* CENTER CONTENT */
+.header-center {
+  grid-column: 2;
+  text-align: center;
+}
+
+/* RIGHT BUTTON */
+.theme-toggle {
+  grid-column: 3;
+  justify-self: end;
+
+  padding: 6px 10px;
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)